playground/terraform/README.md

```diff
@@ -16,74 +16,118 @@
     specific language governing permissions and limitations
     under the License.
 -->
+# Playground deployment on GCP
 
```

Comment:
This guide shows you how to deploy full Playground environment on Google Cloud Platform (GCP) environment.

Reply:
Fixed